### ISSUE TITLE
More Ways to Add Reference

### ISSUE THREAD

**Issue by gitkeniwo** [usememos/memos#3141]
### Describe the solution you'd like

**tldr**: bring back the mark button or the ability to add reference by memo id.

In current version, the only way to add reference *as a link* is by 'searching content'. 

As it turns out, in practice, it is often the case that you may wanna mark certain memo and start writing after some filtering and browsing. It is inconvenient to have to go back to the top and recall the keywords in the memo you would like to mark so that you can search again and locate that memo. 

Would be more intuitive to have the mark button next to the memo like we used to, or being able to add reference by copying the memoid so that you *don't have to figure out how to match the memo content in the search box*.

### Type of feature

User Experience (UX)
